lib/phoenix_live_reload/live_reloader.ex

```diff
@@ -77,7 +76,7 @@ defmodule Phoenix.LiveReloader do
 
         put_in conn.resp_body, body
       else
-        conn
+        put_in conn.resp_body, (resp_body <> missing_tag)
```

Comment:
This branch is also called if the content type is not HTML, so this breaks json responses and so on.